addons/mail/static/src/models/thread/thread.js

```diff
@@ -1089,7 +1150,12 @@ function factory(dependencies) {
         moderation: attr({
             default: false,
         }),
+        moduleIcon: attr(),
```

Comment:
ok for now, but this attr has nothing to do with a thread and not even with a record either...